From ac58a0dd5990cdf2654a055e09457a737ea3c762 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Mon, 28 May 2018 16:05:19 +0800
Subject: [PATCH 0682/1200] arm64: dts: apn806: add "ap_" prefix for spi0 node
 and ic20 node

This patch adds "ap_" prefix for apn806's spi0 node and ic20 node to
distinguish them from CP's corresponding nodes.

Change-Id: Ia042db7930995450e44a7db8f429dc5cd277d3a2
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55760
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-ap806.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/armada-ap806.dtsi b/arch/arm/dts/armada-ap806.dtsi
index 6ed41a4f4d..d0564c0394 100644
--- a/arch/arm/dts/armada-ap806.dtsi
+++ b/arch/arm/dts/armada-ap806.dtsi
@@ -100,7 +100,7 @@
 				#gpio-cells = <2>;
 			};
 
-			spi0: spi@510600 {
+			ap_spi0: spi@510600 {
 				compatible = "marvell,armada-380-spi";
 				reg = <0x510600 0x50>;
 				#address-cells = <1>;
@@ -109,7 +109,7 @@
 				status = "disabled";
 			};
 
-			i2c0: i2c@511000 {
+			ap_i2c0: i2c@511000 {
 				compatible = "marvell,mv78230-i2c";
 				reg = <0x511000 0x20>;
 				#address-cells = <1>;
-- 
2.22.0

